lizard_ext/lizardfans.py

```diff
@@ -57,9 +69,7 @@ def calculate_fan_in_fan_out(name_list, body, fileinfo, func):
             for i, name in enumerate(name_list):
                 re.split(r'; |, |\*|\n', line)
```

Comment:
I need to get acess to your codereader to get the tokens and replace this line of code. Can I call a function to it so it will get rid of the useless information nad make the code cleaner?